Examples/Cocoa/Sources/objc/Pin.m

```diff
@@ -121,6 +123,26 @@ - (instancetype)initWithModelDictionary:(NS_VALID_UNTIL_END_OF_SCOPE NSDictionar
             }
         }
         {
+            __unsafe_unretained id value = modelDictionary[@"tags"]; // Collection will retain.
+            if (value != nil) {
+                if (value != (id)kCFNull) {
+                    NSArray *items = value;
+                    NSMutableArray *result0 = [NSMutableArray arrayWithCapacity:items.count];
+                    for (id obj0 in items) {
+                        if (obj0 != (id)kCFNull) {
+                            id tmp0 = nil;
+                            tmp0 = obj0;
```

Comment:
Can `obj0` ever `nil` in here as we iterate through an `NSArray`

Reply:
Not that I know of.

Reply:
So should we remove this tmp0 copy than and just add the obj0 in the result0 directly?